### ISSUE TITLE
[Suggestion] non-full block conveyor belts

### ISSUE THREAD

**Issue by Zedrimar** [rwtema/extrautilities_old_issues#604]
I love the conveyor belts in this mod, in fact its one of the biggest reasons that I use it among others, I think that it would be nice to be able to choose in the config to have a tile entity conveyor belt similar to MFR's version without the animations of coarse :P maybe something with a rail like texture on the side with an arrow or arrows down the middle so that way no one thinks its a rail and tries to put a minecart on it, maybe have a texture with the same concept like my fail version that I tried to make... (I am a bad artist so bear with me)
![conveyer](https://cloud.githubusercontent.com/assets/8153290/7508766/18e44962-f43b-11e4-94bf-94bdfb58cc67.png)

**Comment by jawsawn** [comment]
I think EXUtilities Conveyor Belts are so underrated. Tema should sparkle dark magic on it :p

**Comment by Zedrimar** [comment]
@JAWSAWN yeah, if not I will use MFR or make a stand alone conveyerbelts mod myself...
